modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro

```diff
@@ -677,6 +677,54 @@ definition {
 		return ${managerId};
 	}
 
+	macro createObjectDefinitionWithAllKindsOfFieldTypes {
+		var objectDefinitionId = ObjectDefinitionAPI.createAndPublishObjectDefinition(
+			en_US_label = ${en_US_label},
+			en_US_plural_label = ${en_US_plural_label},
+			name = ${objectDefinitionName},
+			requiredStringFieldName = ${requiredStringFieldName});
+
+		Picklist.addPicklistViaAPI(picklistName = "customPicklist");
```

Comment:
Can Picklist.addPicklist outsite of the macro createObjectDefinitionWithAllKindsOfFieldTypes?

Reply:
so you mean I move them to .testcase?

Reply:
yes, I'm wondering if it is possible to make existing picklist as a variable to createObjectDefinitionWithAllKindsOfFieldTypes, but this is not required in this pr on my side

Reply:
but we have set the constant in createObjectEntryWithAllFields 'customPicklist' unless we modify all the constants in createObjectEntryWithAllFields

Reply:
@Hijky maybe we could take it out as Ashley says, and then do the if (isSet(picklist)) in the createObjectDefinitionWithAllKindsOfFieldTypes